Cargo.toml

```diff
@@ -28,7 +28,7 @@ cw20 = "1.0.1"
 cw20-base = { version = "1.1.0", features = ["library"] }
 cw-storage-plus = "1.1.0"
 cw-utils = "1.0.1"
-protobuf = { version = "3.2.0", features = ["with-bytes"] }
+protobuf = { version = "=3.2.0", features = ["with-bytes"] }
```

Comment:
I thought so too! Apparently omitting it equals `^`, which is basically means:

`1.2.3  :=  >=1.2.3, <2.0.0`

So the minimum version becomes 3.2.0 in our case, but it can go up to 4.0.0 (excluding). I remember we got issues with protobuf in the past because suddenly some new version appeared. The `=` forces the crate (and its dependencies) to use that version even if another package could pull a newer version.

See this for reference: 
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies